Uppgift12-Garage20/Controllers/ParkedVehiclesController.cs

```diff
@@ -1,157 +1,167 @@
-
-        private bool ParkedVehicleExists(int id)
-        {
-            return _context.ParkedVehicle.Any(e => e.ParkedVehicleId == id);
-        }
-    }
-}
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Mvc.Rendering;
+using Microsoft.EntityFrameworkCore;
+using Uppgift12_Garage20.Data;
+using Uppgift12_Garage20.Models;
+
+namespace Uppgift12_Garage20.Controllers
+{
+    public class ParkedVehiclesController : Controller
+    {
+        private readonly GarageContext _context;
+
+        public ParkedVehiclesController(GarageContext context)
+        {
+            _context = context;
+        }
+
+        // GET: ParkedVehicles
+        public async Task<IActionResult> Index()
+        {
+            return View(await _context.ParkedVehicle.ToListAsync());
+        }
+
+        // GET: ParkedVehicles/Details/5
+        public async Task<IActionResult> Details(int? id)
+        {
+            if (id == null)
+            {
+                return NotFound();
+            }
+
+            var parkedVehicle = await _context.ParkedVehicle
+                .FirstOrDefaultAsync(m => m.ParkedVehicleId == id);
+            if (parkedVehicle == null)
+            {
+                return NotFound();
+            }
+
+            return View(parkedVehicle);
+        }
+
+        /// <summary>
+        /// Displays the view for creating a new parked vehicle.
+        /// </summary>
+        /// <returns>The view for creating a new parked vehicle.</returns>
+        public IActionResult Create()
+        {
+
+            ViewBag.VehicleTypes = new SelectList(Enum.GetValues(typeof(VehicleType)));
+
+            return View();
+
+        }

+
+        /// <summary>
+        /// Creates a new parked vehicle.
+        /// </summary>
+        /// <param name="parkedVehicle">The parked vehicle to create.</param>
+        /// <returns>The action result.</returns>
+        [HttpPost]
+        [ValidateAntiForgeryToken]
+        public async Task<IActionResult> Create([Bind("ParkedVehicleId,VehicleType,RegistrationNumber,Color,Make,Model,NumberOfWheels,ArrivalTime")] ParkedVehicle parkedVehicle)
+        {
+            if (ModelState.IsValid)
+            {
+                _context.Add(parkedVehicle);
+                await _context.SaveChangesAsync();
+                return RedirectToAction(nameof(Index));
+            }
+            return View(parkedVehicle);
+        }
+
+        // GET: ParkedVehicles/Edit/5
+        public async Task<IActionResult> Edit(int? id)
+        {
+            if (id == null)
+            {
+                return NotFound();
+            }
+
+            var parkedVehicle = await _context.ParkedVehicle.FindAsync(id);
+            if (parkedVehicle == null)
+            {
+                return NotFound();
+            }
+            ViewBag.VehicleTypes = new SelectList(Enum.GetValues(typeof(VehicleType)));

```

Comment:
Here you are adding VehicleTypes to the ViewBag  but later on in the Edit View you are not using it. 
Either delete this line or use it in the Edit View.